src/components/ScoreSetHeatmap.vue

```diff
@@ -318,14 +319,24 @@ export default {
             .padding(0.05)
         if (showYTickMarks) {
           svg.append('g')
-              .style('font-size', 15)
+              .attr('class', 'mave-heatmap-y-axis-tick-labels')
               .call(
                 d3.axisLeft(yScale)
                     .tickSize(0)
-                    .tickFormat(n => AMINO_ACIDS_BY_HYDROPHILIA[20 - n])
-                    // .tickFormat(n => String.fromCharCode('A'.charCodeAt(0) + n - 1))
+                    .tickFormat((n) => {
+                      // Get the row's amino acid code or variation symbol.
+                      const label = HEATMAP_ROWS[HEATMAP_ROWS.length - 1 - n]
```

Comment:
repeating this logic to reverse the HEATMAP_ROWS gets very confusing to the reader.

I think there are a few options here. 

1) reverse heatmap_rows
2) use a d3 scalePoint as in this notebook to combine the flipping and translation of letter to axis position: https://observablehq.com/@mbostock/phases-of-the-moon
3) to eliminate the backwards iteration below, use a data structure for each heatmap_rows entry with style and label attributes that have defaults, so we have entries like {'symbol': '-', 'style': 'mave-heatmap-y-axis-tick-label-lg'} and {'symbol': '*', 'label': '\uff0a'}